tests/test_velocity_verlet_integrator.py

```diff
@@ -0,0 +1,142 @@
+from functools import partial
+
+import numpy as np
+
+from timemachine.integrator import VelocityVerletIntegrator as ReferenceVelocityVerlet
+from timemachine.lib import VelocityVerletIntegrator, custom_ops
+from timemachine.lib.potentials import SummedPotential
+from timemachine.testsystems.relative import hif2a_ligand_pair
+
+
+def setup_velocity_verlet(bps, x0, box, dt, masses):
+    integrator = VelocityVerletIntegrator(dt, masses)
+    # return integrator impl to avoid deallocating
+    intg = integrator.impl()
+    context = custom_ops.Context(x0, np.zeros_like(x0), box, intg, bps)
+    return intg, context
```

Comment:
Would it be worth adding a wrapper for `Context` to the Python API to avoid needing this workaround? Something like

```python
class Context(custom_ops.Context):
    def __init__(self, x0, v0, box, integ, bps):
        self._integ = integ
        super().__init__(x0, v0, box, integ, bps)
```
and then
```suggestion
    return custom_ops.Context(x0, np.zeros_like(x0), box, integrator.impl(), bps)
```

Reply:
Makes sense to me, will give that a shot.

Reply:
Created an issue for this https://github.com/proteneer/timemachine/issues/791 as it seems out of scope for this PR (especially since this is a test function) and is a broader problem we should resolve.